src/rebar_prv_gradualizer.erl

```diff
@@ -17,6 +17,9 @@
 -define(PROVIDER, gradualizer).
 -define(DEPS, [compile]).
 
+%% This is borrowed from the private upstream rebar.hrl
+-define(DEBUG(Str, Args), rebar_log:log(debug, Str, Args)).
```

Comment:
Does this give debugging printouts with some command line option, like `rebar3 --debug gradualizer` or something?

Reply:
Yes, that's what the `DEBUG=1 ...` from the PR description does.

Reply:
Ahh 🤦‍♂️PR description, too obvious